src/search.rs

```diff
@@ -50,8 +50,8 @@ pub struct SearchResult<T> {
 #[derive(Deserialize, Debug, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct FacetStats {
-    pub min: u32,
-    pub max: u32,
+    pub min: f64,
+    pub max: f64,
```

Comment:
does `u64` not fix this issue? I don't think facet stats can return a negative number

Reply:
`u64` is an unsigned integer so according to issue it would still error with this since it needs a decimal.

Reply:
Oh yeah ... my bad 🙈

Reply:
Looked in meilisearch repo, and there this struct has f64 fields https://github.com/search?q=repo%3Ameilisearch%2Fmeilisearch%20FacetStats&type=code